x-pack/plugins/security_solution/common/endpoint/data_generators/exceptions_list_item_generator.ts

```diff
@@ -250,4 +251,70 @@ export class ExceptionsListItemGenerator extends BaseDataGenerator<ExceptionList
       ...overrides,
     };
   }
+
+  generateBlocklist(overrides: Partial<ExceptionListItemSchema> = {}): ExceptionListItemSchema {
+    return this.generate({
+      name: `Blocklist ${this.randomString(5)}`,
+      list_id: ENDPOINT_BLOCKLISTS_LIST_ID,
+      item_id: `generator_endpoint_blocklist_${this.randomUUID()}`,
```

Comment:
can you use the `.seededUUIDv4()` method instead so that we get deterministic output based on the seed used to insatiate the class instance